### PR TITLE
[SMALL] Adding tests for the #2069 - Order by Guid.NewGuid() returns same order every time

### PR DESCRIPTION
**Pull request by maumar**
Also porting OrderDetail data from Northwind so that the results match our baselines.

### CODE REVIEW

**Comment by maumar** [comment]
@anpete

**Comment by anpete** [comment]
@maumar Reminder, I think you need to do something so that you show up as "cla-not-requred".

File: test/EntityFramework.Core.FunctionalTests/QueryTestBase.cs [hunk 1]
```diff
@@ -3535,6 +3540,14 @@ protected QueryTestBase(TFixture fixture)
         {
             using (var context = CreateContext())
             {
+
+                var foo = NorthwindData.Set<TItem>().Count();
```
Comment (anpete) [inline test/EntityFramework.Core.FunctionalTests/QueryTestBase.cs]: ??

**Comment by anpete** [comment]
:shipit:

**Pull request merged**